Update README: feature list, document hyperlinks and getting started guide (#247)
skypyproject/skypy: README.rst

.. _Gravitational Wave: https://skypy.readthedocs.io/en/latest/gravitational_wave/index.html
.. _Power Spectra: https://skypy.readthedocs.io/en/latest/power_spectrum/index.html
.. _Pipelines: https://skypy.readthedocs.io/en/latest/pipeline/index.html
.. _SkyPy Documentation: https://skypy.readthedocs.io/en/latest/
.. _Citation Guidelines: CITATION

Getting Started
---------------

SkyPy is distributed through PyPI_ and conda-forge_. To install SkyPy and its
dependencies_ using pip_:

.. code:: bash

    $ pip install skypy

To install using conda_:

.. code:: bash

    $ conda install -c conda-forge skypy

You can test your SkyPy intallation using pytest_:

.. code:: bash

    $ pytest --pyargs skypy

The SkyPy library can then be imported from python:

.. code:: python

    >>> import skypy
    >>> help(skypy)

SkyPy also has a driver script that can run pipelines. The repository contains
examples_ of config files in yaml format to download and run. For example:

.. code:: bash

    skypy –config examples/herbel_galaxies.yaml –format fits

will generate tables of red and blue galaxies following the model described in

> `Herbel et al. 2017`_ and write them in fits format.

Reviewer: ```suggestion
`Herbel et al. 2017`_ and write them in FITS format.
```

Way. Also included are methods for generating observed data.

Summary of SkyPy features
-------------------------

> The full list of features can be found on the
> [SkyPy Documentation](https://skypy.readthedocs.io/en/latest/?badge=latest) main page
> * Clusters
> * Galaxies
> * Gravitational waves
> * Dark Matter Halos
> * Power Spectrum
> * Supernovae
> * Utiles
> * Pipeline

Reviewer 2: ```suggestion
The full list of features can be found on the
[SkyPy Documentation](https://skypy.readthedocs.io/en/latest/?badge=latest) and includes:

* Clusters
* Galaxies
* Gravitational Waves
* Dark Matter Halos
* Power Spectrum
* Supernovae
* Utilities
* Pipeline
```

Important links
---------------

> * [License](https://github.com/skypyproject/skypy/blob/master/LICENSE.rst)
> * [Code of Conduct](https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md)
> * [Contributing guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
> * [Citation guidelines](https://github.com/skypyproject/skypy/blob/master/CITATION)

Reviewer 2: ```suggestion
* `License <https://github.com/skypyproject/skypy/blob/master/LICENSE.rst>`_
* `Code of Conduct <https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md>`_
* `Contributor Guidelines <https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)>`_
* `Citation Guidelines <https://github.com/skypyproject/skypy/blob/master/CITATION>`_
```

Way. Also included are methods for generating observed data.

Summary of SkyPy features
-------------------------

The full list of features can be found on the
`SkyPy Documentation <https://skypy.readthedocs.io/en/latest/?badge=latest>`_ main page

* Clusters
* Galaxies
* Gravitational waves
* Dark Matter Halos
* Power Spectrum
* Supernovae
* Utiles
* Pipeline

Important links
---------------

* `License <https://github.com/skypyproject/skypy/blob/master/LICENSE.rst)>`_

> * `Code of Conduct <https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md)>`_
> * `Contributing guidelines <https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)>`_
> * `Citation guidelines <https://github.com/skypyproject/skypy/blob/master/CITATION)>`_

Reviewer 2: ```suggestion
* `Code of Conduct <https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md>`_
* `Contributor Guidelines <https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md>`_
* `Citation Guidelines <https://github.com/skypyproject/skypy/blob/master/CITATION>`_
```

Author: Good spot! Thanks. This is fixed now.

How to install and run SkyPy
----------------------------

* Requirements:

  Make sure you have current version of ``astropy``, ``networkx``,
  ``numpy``, ``scipy`` and ``pyyaml``.

* Installation with pip:

  ``pip install skypy``

* Installation with conda:

>   (coming soon)

Reviewer 2: ```suggestion
  ``conda install -c conda-forge skypy``
```

* Running tests:

>   ``python setup.py test``

Reviewer 2: ```suggestion
  ``pytest --pyargs skypy``
```
